torch_geometric/nn/__init__.py

```diff
@@ -1,18 +1,20 @@
+from .data_parallel import DataParallel
```

Comment:
`isort` is disabled on `__init__.py`, so we do not necessarily need this. Doesn't hurt though.